### PR TITLE
Add binary support

### PR DESCRIPTION
**Pull request by avifenesh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: redis/src/commands/cluster_scan.rs [hunk 1]
```diff
@@ -487,7 +487,7 @@ where
 async fn send_scan<C>(
     scan_state: &ScanState,
     core: &C,
-    match_pattern: Option<String>,
+    match_pattern: Option<Vec<u8>>,
```
Comment (eifrah-aws) [inline redis/src/commands/cluster_scan.rs:490]: `BytesMut` is more efficient, any special reason why you choose `Vec<u8>` ? (`BytesMut` is similar to `Vec<u8>` but with less copies and allocations)

Comment (avifenesh) [reply redis/src/commands/cluster_scan.rs:490]: The built in function of the library is to_redis_args which return Vec<u8>

Comment (avifenesh) [reply redis/src/commands/cluster_scan.rs:490]: Might be better to refactor but not at this point

Comment (eifrah-aws) [reply redis/src/commands/cluster_scan.rs:490]: Makes sense not to make a drastic change at this point

**Review by eifrah-aws — APPROVED** [review]

**Review by eifrah-aws — APPROVED** [review]

**Pull request merged**